synapse/lib/stormlib/storm.py

```diff
@@ -13,6 +15,57 @@
 
 stormlogger = logging.getLogger('synapse.storm')
 
+class StormExecCmd(s_storm.Cmd):
+    '''
+    Execute text or an embedded query object as Storm in the current pipeline.
+
+    NOTE: It is recommended to avoid using this where possible to avoid potential
+    query injection risks. If you must use this, take care to ensure any values
+    being executed have been properly sanitized.
+
+    Examples:
+
+        // Add nodes using text in a variable
+        $query = '[ inet:fqdn=foo.com inet:fqdn=bar.net ]'
+        storm.exec $query
+
+        // Filter nodes in the pipeline using text in a variable
+        $filter = '-:asn=10'
+        inet:ipv4:asn
+        storm.exec $filter
+
+        // Pivot using an embedded query
+        $pivot = ${ -> inet:asn }
+        inet:ipv4:asn
+        storm.exec $pivot
+    '''
+    name = 'storm.exec'
+    def getArgParser(self):
+        pars = s_storm.Cmd.getArgParser(self)
+        pars.add_argument('query', help='The Storm to execute.')
+        return pars
+
+    async def execStormCmd(self, runt, genr):
+
+        item = None
+        async for item in genr:
+
+            text = await s_stormtypes.tostr(self.opts.query)
+            query = await runt.getStormQuery(text)
+
+            async with runt.getSubRuntime(query) as subr:
```

Comment:
I'd like to make these pump through one runtime so it behaves more like it's inline. Thoughts?

Reply:
Yeah I was going back and forth on that, I can switch it to one runtime. Do we want to allow non-runtsafe queries and potentially run different storm per-node?

Reply:
Yea, unfortunately we probably need both.

Reply:
I adjusted it to use a single runtime and just swap out the query per-node